common/rfb/CConnection.h

```diff
@@ -145,7 +145,12 @@ namespace rfb {
     // case no user name will be retrieved.
     virtual void getUserPasswd(bool secure, std::string* user,
                                std::string* password) = 0;
-    
+    /*!
+     * Get x509 authentication CA and CRL file, the file format is pem.
+     * \param ca: certificate authority, the file format is pem.
+     * \param crl: certificate revocation list, the file format is pem.
+     */
+    virtual int getX509File(std::string* ca, std::string* crl) = 0;
```

Comment:
This seems very unrelated to this PR?

Reply:
Yes. Another similar issue, so I'll put it together.
 
CSecurityTLS: change the variable that x509 authentication CA and CRL file from global to local

Reply:
I'm afraid I don't see them as similar. Please open a new PR with a rationale for why this change is needed.